pkg/test/controller/k8s.go

```diff
@@ -160,7 +160,9 @@ func ReplaceTestVars(t *testing.T, b []byte, uniqueId string, project testgcp.GC
 	s = strings.Replace(s, fmt.Sprintf("projects/${%s}", testgcp.TestDependentOrgProjectId), fmt.Sprintf("projects/%s", testgcp.GetDependentOrgProjectID(t)), -1)
 	s = strings.Replace(s, fmt.Sprintf("${%s}", testgcp.TestDependentOrgProjectId), fmt.Sprintf("\"%s\"", testgcp.GetDependentOrgProjectID(t)), -1)
 	s = strings.Replace(s, fmt.Sprintf("projects/${%s}", testgcp.TestDependentFolderProjectId), fmt.Sprintf("projects/%s", testgcp.GetDependentFolderProjectID(t)), -1)
+	s = strings.Replace(s, fmt.Sprintf("projects/${%s}", testgcp.TestDependentFolder2ProjectId), fmt.Sprintf("projects/%s", testgcp.GetDependentFolder2ProjectID(t)), -1)
 	s = strings.Replace(s, fmt.Sprintf("${%s}", testgcp.TestDependentFolderProjectId), fmt.Sprintf("\"%s\"", testgcp.GetDependentFolderProjectID(t)), -1)
+	s = strings.Replace(s, fmt.Sprintf("${%s}", testgcp.TestDependentFolder2ProjectId), fmt.Sprintf("\"%s\"", testgcp.GetDependentFolder2ProjectID(t)), -1)
```

Comment:
Nit: Am I missing something, or do we not need the previous replacement if we have this?  Anyway, you're following the pattern so that's a question for another day!

Reply:
We may not need the replacement. Just to follow the pattern.